www/_posts/2018-07-11-cordova-android-7.1.1.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+author:
+    name: Chris Brody
+    url: https://twitter.com/brodybits
+title:  "Cordova Android 7.1.1 Released!"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce that `Cordova Android 7.1.1` has been released! This release fixes various bugs releated to the previous `cordova-android@7.1.0` release.
+
+To upgrade:
+
+    npm install -g cordova
+    cd my_project
+    cordova platform remove android
+    cordova platform add android@7.1.1
+
+To add it explicitly:
+
+    cordova platform add android@7.1.1
+
+<!--more-->
+
+## Curated Changelog
+
+* Fix unsafe property access in run.js (#445)
+* Emit log event instead of logging directly (#452)
+* [CB-14101](https://issues.apache.org/jira/browse/CB-14101) Fix Java version check for Java >= 9 (#446)
```

Comment:
We should be consistent in how/when we link to tickets. `(#446)` is meaningless on its own at the end of a line here.

*If* we want to include links to GitHub PRs/issues, they should be formatted and linked as per GH-#### convention:
`[GH-446](https://github.com/apache/cordova-android/pull/446): <description>`

If we have both a Jira ticket and a PR, we should link to Jira because the PR changes should be linked from there

Reply:
Done, thanks!